pkg/block/fetcher.go

```diff
@@ -98,8 +102,10 @@ const (
 	// Modified label values.
 	replicaRemovedMeta = "replica-label-removed"
 
-	tenantLabel  = "__tenant__"
-	defautTenant = "__unknown__"
+	tenantLabel    = "__tenant__"
+	defautTenant   = "__unknown__"
+	replicaLabel   = "__replica__"
+	defaultReplica = ""
```

Comment:
shall we not use empty label value? maybe just `n_a`?

Reply:
An empty value label will be removed by M3. It's perfect for our use case.
<img width="1467" alt="image" src="https://github.com/databricks/thanos/assets/129140017/f8836da3-0b65-4a1c-ad2c-d49d0fd1d77d">